### DISCUSSION TITLE
save routes in collaborative maps

### DISCUSSION THREAD

**Discussion by shakebox** [FacilMap/facilmap#315] [Ideas]
hi there!

just came across this fantastic service while searching for tool to plan vacation road trip. it's perfect to store markers for points i want to see, possible camper sites, restaurants, etc.

but when i go from collecting all possible destination points to actually planning a route (or compare different possible routes), there is one thing i'm missing at the moment: to save the route that i set up. NOT the actual resulting path. i can easily save this as a line. but the line is then fixed and can't changed back to a route where i add additional waypoints or delete a waypoint etc.

so having the option to save the current setup waypoints of a route in combination with the chosen parameters (route type, shortest/fastest, etc.) in a similar way that you can save views already, that would be really helpful. would allow you to easily switch between different saved routes, compare them, change them afterwards, etc. and also store them between sessions of usage of the map. because right now if i close the browser tab/window and open it later again with one of the 3 links (admin, etc.), i only get my markers and lines back, not my routes, right?

setup could be very similar to views: 'save route' in the route area. and then a list of saved routes to load one. and a 'manage routes' area to sort and delete them.

thx for looking into this!

**Comment by cdauth** [comment]
Have you tried clicking “Actions” → “Edit waypoints” on the line? That should do what you are looking for, unless I’m misunderstanding you.

**Reply by shakebox** [reply]
oh, you are right. I used the german version of the site and "Bewegen" (german for 'move') didn't give me the hint that this could bring me to a full edit mode. but now with your hint I also found the related entry in the documentation.

so please close this idea, it's already integrated in your tool. only way to improve it would be to save the 'names' of the waypoints as well when saving a line. somehow related to my other idea https://github.com/FacilMap/facilmap/discussions/316 (which would better have been an 'issue', right?). but I understand the logic behind it, so maybe just not possible to save these 'names' of a waypoint, as waypoints are just coordinates, right?
